// Utils provides different function for handling errors, helper functions
const catchAsync = require('./catchAsync');

module.exports = {
  catchAsync,
};
